day_6/exercises/practice.rb

```diff
@@ -0,0 +1,30 @@
+class GoodDog
+  attr_accessor :name, :height, :weight
+
+  def initialize(n, h, w)
+    @name = n
+    @height = h
+    @weight = w
+  end
+
+  def speak
+    "#{name} says Arf!"
+  end
+
+  def change_info(n, h, w)
```

Comment:
It can seem tempting to make multi purpose methods like this, but I'd try to go for single purpose methods. Nothing is wrong here, its just that everytime you want to change one of those things, you need to input all 3.